dev-requirements.txt

```
@@ -1,16 +1,16 @@
 black==24.10.0; python_version > "3.8"
-black==24.8.0; python_version <= "3.8"
+black==24.10.0; python_version <= "3.8"
```

```suggestion
black==24.8.0; python_version <= "3.8"
```
Reverting because this is the last version of black that supports 3.8